client/src/pages/component/homeBody/Homebody.css

```diff
@@ -1,4 +1,84 @@
-.homebody{
-    text-align: center;
-    margin-top: 100px;
+.homebody-container {
+	background-image: url(../image/home-body.jpg);
+	background-repeat: no-repeat;
+	width: 100%;
+	height: 650px;
+	background-size: contain;
+}
+
+.body-main-card {
+	max-width: 1000px;
+	width: 100%;
+	margin: auto;
+	height: 60vh;
+	text-align: center;
+	align-items: center;
+	justify-content: center;
+	position: absolute;
+	right: 0;
+	left: 0;
+	top: 0;
+	bottom: 0;
```

Comment:
Here you have a few properties from flex layout (but you missed `display: flex`) and a few properties from absolute positioning. I would suggest you to decide wich way you want to position your elements  - flexbox or absolute positioning and stick to it